projects/chaos/src/emergency_fault.c

```diff
@@ -45,6 +45,11 @@ static StatusCode prv_ack_handler(CANMessageID msg_id, uint16_t device, CANAckSt
   return STATUS_CODE_OK;
 }
 
+void emergency_fault_init_storage(EmergencyFaultStorage *storage) {
```

Comment:
You can probably just call this `_init`.

Reply:
Done.